### PR TITLE
Base docker container image configuration for Laravel APIs

### PR DESCRIPTION
**Pull request by jeurrutia**

### CODE REVIEW

File: rootfs/start.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -z ${EXECUTE_DEV_INSTALL+x} ]; then
+    composer install --no-interaction
+    composer clear-cache
+fi
+if [ ! -z ${EXECUTE_LARAVEL_OPTIMIZATION+x} ]; then
```
Comment (jeurrutia) [inline rootfs/start.sh]: I added a new `ENV` variable to optimize Laravel code when we deploy in a production environment. For dev environment we can omit it

File: rootfs/usr/local/etc/php/php.ini [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+unserialize_callback_func =
+serialize_precision = 17
+disable_functions =
+disable_classes =
+zend.enable_gc = On
+expose_php = On
+max_execution_time = 30
+max_input_time = 60
+memory_limit = 256M
+error_reporting = Off
```
Comment (bvis) [inline rootfs/usr/local/etc/php/php.ini]: Wouldn't this hide different error levels?

File: rootfs/usr/local/etc/php/php.ini [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+[Tidy]
+tidy.clean_output = Off
+[soap]
+soap.wsdl_cache_enabled=1
+soap.wsdl_cache_dir="/tmp"
+soap.wsdl_cache_ttl=86400
+soap.wsdl_cache_limit = 5
+[ldap]
+ldap.max_links = -1
+error_log = /var/log/php-error_log
```
Comment (bvis) [inline rootfs/usr/local/etc/php/php.ini]: I think it's better to log the errors to STDERR instead

Comment (jeurrutia) [reply rootfs/usr/local/etc/php/php.ini]: Yes. I'll modify it!

**EDIT** Modified!

File: rootfs/start.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    php artisan config:cache
+    php artisan route:cache
+    php artisan vendor:publish --tag=public
+fi
+
+if [ ! -z ${EXECUTE_MIGRATIONS+x} ]; then
+    php artisan migrate:refresh --seed
+fi
```
Comment (akira28) [inline rootfs/start.sh]: As talked before, EXECUTE_MIGRATIONS should be renamed to EXECUTE_REFRESH and add an `else` statement containing `php artisan migrate --force`

File: rootfs/usr/local/etc/php/conf.d/30-xdebug.ini [hunk 5]
```diff
@@ -0,0 +1,10 @@
+;zend_extension = xdebug.so
```
Comment (akira28) [inline rootfs/usr/local/etc/php/conf.d/30-xdebug.ini]: Maybe we should find a way to load this ini just on dev, so we don't have to uncomment this

File: rootfs/start.sh (hunk 4 above)
Comment (jeurrutia) [reply rootfs/start.sh]: Ok. I'll do the changes

File: rootfs/usr/local/etc/php/conf.d/30-xdebug.ini (hunk 5 above)
Comment (jeurrutia) [reply rootfs/usr/local/etc/php/conf.d/30-xdebug.ini]: Yup. For dev environments we mount a volume with specific configuration values.

**Review by akira28 — APPROVED** [review]

File: rootfs/usr/local/etc/php/php.ini (hunk 2 above)
Comment (jeurrutia) [reply rootfs/usr/local/etc/php/php.ini]: Changed to `E_ALL`

**Comment by bvis** [comment]
I'm missing README.md and LICENSE files

**Comment by jeurrutia** [comment]
README.md is already in the repository.

**Pull request merged**